dockerfiles/theia-dev/docker/ubi8/setup-yarn.dockerfile

```diff
@@ -1,2 +1,2 @@
 # install yarn dependency
-RUN npm install -g yarn@1.17.3
+RUN npm install -g yarn@1.21.1
```

Comment:
fyi @nickboldt 
maybe it affects crw build

Reply:
yes, obviously this affects CRW. Thanks for the headsup (which I'm only seeing now) :(